packages/react-api/src/hooks/useCartoLayerProps.js

```diff
@@ -28,25 +31,32 @@ export default function useCartoLayerProps({
 
   let props = {};
 
-  if (
+  const useMVT =
```

Comment:
In c4r and cloud-native we try to don't use `MVT` for naming. Better useTileset

Reply:
Agree. Updated.